### PR TITLE
Sconify with latest scone tooling (v5.3.3)

### PR DESCRIPTION
**Pull request by zguesmi**

### CODE REVIEW

File: src/test/resources/run/session.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-name: 1
-version: "0.3"
+name: 'sessionId'
```
Comment (jeremyjams) [inline src/test/resources/run/session.yml:1]: Minor: I would use something like "currentSession".

File: src/test/resources/run/session.yml [hunk 2]
```diff
@@ -12,20 +12,39 @@ access_policy:
...
+  
+  - name: 'pre-compute'
+    image_name: 'pre-compute-image'
+    mrenclaves: ['a45df00514ba45be07d322afb33d55cb4183adfc4e174098c365491f3af073e2']
+    pwd: '/'
+    command: 'java -jar /app/app.jar'
+    environment:
+      LD_LIBRARY_PATH: '/usr/lib/jvm/java-11-openjdk/lib/server:/usr/lib/jvm/java-11-openjdk/lib:/usr/lib/jvm/java-11-openjdk/../lib'
+      JAVA_TOOL_OPTIONS: '-Xmx256m'
+      IEXEC_TASK_ID: 'taskId'
```
Comment (jeremyjams) [inline src/test/resources/run/session.yml:24]: Do we need these IEXEC_* for local tests?

**Review by jeremyjams — APPROVED** [review]

File: src/test/resources/run/session.yml (hunk 2 above)
Comment (zguesmi) [reply src/test/resources/run/session.yml:24]: No, they are not mandatory. If values are not set up, a runtime error (Dataset download failed for e.g.) will be thrown. But with those values, we can test the full workflow.

File: src/test/resources/run/session.yml (hunk 1 above)
Comment (zguesmi) [reply src/test/resources/run/session.yml:1]: No problem.

**Pull request merged**